static/src/js/actions/project.js

```diff
@@ -100,14 +100,16 @@ export const getProjectGranules = () => (dispatch, getState) => {
         dispatch(updateAuthTokenFromHeaders(response.headers))
         dispatch(updateProjectGranules(payload))
       })
-      .catch((e) => {
-        if (e.response) {
-          const { data } = e.response
+      .catch((error) => {
+        dispatch(actions.handleError(error, 'granules'))
+
+        if (error.response) {
```

Comment:
Why is this `catch` block different than other actions? See comment above about action error messages.